docs/learn/universal-receiver/create-receiver-forwarder.md

```diff
@@ -1,89 +1,83 @@
 ---
 sidebar_label: 'Create a LSP1 Forwarder'
 sidebar_position: 2
+description: This smart contract tutorial guides you on how to create a LSP1 Delegate contract that forwards portion of received tokens automatically to any address.
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Creating the LSP1 Forwarder
 
-In this guide, we will create a custom [Universal Receiver Delegate](../../standards/generic-standards/lsp1-universal-receiver-delegate.md) contract. This contract will be called each time the associated UP receives a [LSP7 token](../../standards/tokens/LSP7-Digital-Asset.md), and will forward a certain percentage to another address. The use-case it answers is:
+In this guide, we will create a custom [Universal Receiver Delegate](../../standards/generic-standards/lsp1-universal-receiver-delegate.md) contract. This contract will be called each time the associated UP receives a [LSP7 token](../../standards/tokens/LSP7-Digital-Asset.md), and will forward a certain percentage to another address.
+
+The use-case it answers is:
 
 > **"As a Universal Profile (UP) owner, I want to transfer part of the tokens I received to another UP"**.
 
 An example scenario could be: _"each time I receive USDT, I want to automatically transfer 20% to my wife's UP"_.
 
 ## Requirements
 
-In order to follow this guide, you'll need to:
+:::success Tips
 
-1. Install the [UP Browser extension](/install-up-browser-extension).
-2. Fund the main controller (EOA) of your UP (You can find this address in the extension if you click on the controller tab > "UP Extension") using the [Testnet Faucet](https://faucet.testnet.lukso.network/).
-3. Setup a new Hardhat project.
+If you want to follow this guide using not an existing token, but a new token that you want to create and deploy yourself, check our guide [**"Create a Custom LSP7 Token"**](../digital-assets/smart-contract-developers/getting-started.md#create-a-custom-lsp7-token-contract).
 
-## 1 - EOA permission
+:::
 
-First, we will need to give the EOA that controls our UP proper permission to add / edit an Universal Receiver Delegate (called "Automation").
+In order to follow this guide, you will need the followings:
 
-To do that, click on the controller tab > "UP Extension" which will bring the controller information page.
+1. Install the [UP Browser extension](/install-up-browser-extension).
+2. Fund the main EOA controller of your 🆙 (See **[Step 1](#step-1---enable-your-controller-to-add-a-universal-receiver) bullet point 3** to retrieve its address) using the [Testnet Faucet](https://faucet.testnet.lukso.network/).
+3. The address of the LSP7 token that you want to use to forward of portion of the amount received.
```

Comment:
maybe here as 4. you can add "have read the previous article /deploy-universal-receiver and have setup your development environement" or smtgh

Reply:
I don't think developer will need to have read the previous guide here. Also the other guide might be re-written too.